packages/@tinacms/graphql/src/primitives/resolver/index.ts

```diff
@@ -644,7 +644,9 @@ const resolveDateInput = (
   /**
    * Convert string to `new Date()`
    */
-  const date = parseISO(value)
+  const parsedValue = new Date(value).toISOString()
+  const date = parseISO(parsedValue)
```

Comment:
I only have one concern here.  If `value` is `invalid` for any reason, I think `parsedValue` will fail and could cause the whole resolver to fail?  It is why I originally had the check for `isValid()`.  I would just want to make sure `parsedValue` can fail gracefully (maybe just a `try... catch...` around that)?

Reply:
the problem with a try-catch when the format is bad is, what do you do to recover? It honestly makes sense for the backend to error out here to avoid data corruption

Reply:
So what kind of concern should the parsing be though?

Right now, the DateFieldPlugin is handling the formatting of the date, and then we're essentially passing bad data to the backend. So should the backend have a check to convert say a string to a Date, or should the backend fail, and then it shifts to a client end concern for converting the data.

Reply:
The backend is already parsing a string into a Date, via `parseISO`. What is currently in question is whether we want to be more generous about which date formats the backend accepts (relying on `Date.parse` to figure it out.) If the backend receives a date string that `Date.parse` doesn't know how to parse, we're still left figuring out how to deal with the error.

Reply:
I feel like the backend should only accept UTC, and we need to deal with it on the frontend. I was probably worried too much about how the data is stored when Chris and I originally discussed this

Reply:
@jeffsee55 I agree that the backend should only be accepting UTC, but when talking about addressing this on the frontend are you meaning this transformation occur in local state in `DateFieldPlugin.tsx` or should that be a concern on a per app basis. e.g. a developer is addressing this on a per component basis?